roles/disaster_recovery/README.md

```diff
@@ -0,0 +1,93 @@
+oVirt Disaster Recovery
+=========
+
+The `oVirt.disaster-recovery` role responsible to manage the disaster recovery scenarios in oVirt.
+
+Requirements
+------------
+
+ * Ansible version 2.9 or higher
+ * Python SDK version 4.4 or higher
```

Comment:
There are global requirements in the main README.md. You can remove these lines.